internal/report/report.go

```diff
@@ -21,21 +23,38 @@ type ClusterPolicyReport struct {
 	v1alpha2.ClusterPolicyReport
 }
 
-// Status specifies state of a policy result
 const (
+	// Status specifies state of a policy result
 	StatusPass  = "pass"
 	StatusFail  = "fail"
 	StatusWarn  = "warn"
 	StatusError = "error"
 	StatusSkip  = "skip"
+
+	// Severity specifies severity of a policy result
+	SeverityCritical = "critical"
+	SeverityHigh     = "high"
+	SeverityMedium   = "medium"
+	SeverityLow      = "low"
+	SeverityInfo     = "info"
+
+	// Category specifies the category of a policy result
+	CategoryMutating   = "mutating & validating"
```

Comment:
bit of a mouthful. Better `mutating then validating`, `mutating+validating`, or just `mutating`?

Reply:
After seeing it on policy-reporter filtering (see https://github.com/kubewarden/audit-scanner/pull/47#discussion_r1221453801), I think it should be `mutating`.

Reply:
yes, `mutating` should be enough

Reply:
changed to `mutating` in https://github.com/kubewarden/audit-scanner/pull/47/commits/d39bafd678a5f99b70618dae6842dcee8a8617e0.